tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -223,205 +253,138 @@ async def test_wedge_command_where_noops_should_be_sent_in_two_parts(self, bft_n
     @unittest.skip("manual testcase - not part of CI")
     @with_trio
     @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
-    async def test_semi_manual_upgrade(self, bft_network):
-        """
-             Sends a wedge command and check that the system stops from processing new requests.
-             Note that in this test we assume no failures and synchronized network.
-             The test does the following:
-             1. A client sends a wedge command
-             2. The client then sends a "Have you stopped" read only command such that each replica answers "I have stopped"
-             3. Apollo stops all replicas
-             4. Apollo starts all replicas
-             5. A client verifies that new write requests are being processed
-         """
+    async def test_get_latest_prunebable_block(self, bft_network):
```

Comment:
Nit: wherever applies, change `prunebable` to `pruneable`.

Reply:
done